web/src/assets/svg/logo-Tilt-Cloud.svg

```diff
@@ -0,0 +1 @@
HkBAvSaHyIKRAABBBBAAAEEEEAAgaIKdP6mr9tV1txRcdqf9ljvN5N+Vb9JLW4UJz1OXKWovdMXAmUX+Gw/chtRteFo+nFadn0oSPioGm0ouk+HJh45PlZ2I/pHAAEEEEAAAQQQQAABBFZDgAB9NdR5JgIIIIAAAggggAACCCCwBIG25ze2Vzrb0tLw0u+jdkexDZdnsKvemQL2tiXchlMQQGAVBKKEOTM3enkGea5jPpPReZsfCnk2NP7mkWF5RvJVKI1HIoAAAggggAACCCCAAAIIXEOAAP0aOBxCAAEEEEAAAQQQQAABBGpa4Jdbbl9Xib2VhuY+H6UnauxW8RtVpVss9KZv+Hqdy9bUdA8Uh0AdCqTZ4+fUZCSajQSx0fRv7YRLYXlaYv1kdWH2xNzszInpPSOTddgaJSOAAAIIIIAAAggggAACpRcgQC/9lwAACCCAAAIIIIAAAgggUGiBF9beuqGlq0+z0KPmehrE9VW963LRNpjqehPpciLdaQn5lkI70BwCSxAIFi5E0TGndkqijKeAfNScHw9WPWFmJ11wo2OTZ4blqfELS7gdpyCAAAIIIIAAAggggAACCNShAAF6HQ4aJSOAAAIIIIAAAggggAACN1wgBe09t9ze7ayhKzbYegu2oaJuXRTXYxrXOdHONOO2y9Q6vfpbbvjzuSECKySwGIqn/cXPiOp4CsEngsTTotlJk+ppVT3lgxsLcf70yU+mThKMr9AgcFsEEEAAAQQQQAABBBBAoI4ECNDraLAoFQEEEEAAAQQQQAABBBCoCYFnu1u6u9s7vdN11fRHNKaPXadGWZdmtK/zqp3O9HYT6xAn7SLakfZrr9RE7RRR1wJp6fSqiE2m2eFTKjoZ1T5RsTMW5XTwckainQlRJyqW3ldlfHTh7ITsGZmt66YpHgEEEEAAAQQQQAABBBBA4KYKEKDfVG4ehgACCCCAAAIIIIAAAgiUVOC5ztYN69d25JK3O23oyMR3RGftaZb77dFie1o2uyNzri3t3X5b2sc97duua9LM9zVqtsapby2pWiHbjhbOp+0DpiWmfcSdTJv5aVE5F2M8m8LxqbSlwGRKySd9CsirMUxFW5isxIbJ0dPjU/L0xPlCotAUAggggAACCCCAAAIIIIBAzQgQoNfMUFAIAggggAACCCCAAAIIIIDAVQS07fmNtzW3tLRlTSlQl6Y1CxLXOLPWRp+1BRdbQh6b03LcKYCXlrRUd5Nz7rbcYnP6vKlRpT2YNYv6JpGYZsSLpGXoL76/yvN4+XOBtPz5VDI0UXfWic2m2d5z8yZTKfCey9TNXgy9VWbFbNalc8TZnI9uZj7k56Lq+QZx01HmpvM5nZ6dmTl37ifDKSRP9+MNAQQQQAABBBBAAAEEEEAAgRoVIECv0YGhLAQQQAABBBBAAAEEEEAAgZUXaH+hvy1Wc3fLmtbWoC6raPWWuazS0FzVlpBVG1UbmtIS9c2LlWiwxorLWi5XFWNbUEsTphffXGNaXvzysbS8eFtayv7zY5+d8WV/B7H0DHf5usVz0nLkF7zawpedf+Vr6ZcCYsq2z116LS2TP5N+QWB+8XOfpu+Lc5ePVUO4YJl8dk9zs2YLc5k1zM24ONuUVxeqVrngLeYXps+fd5UsTu0ZunztpfvzHgEEEEAAAQQQQAABBBBAAIEyCPwfyYMYqN30liEAAAAASUVORK5CYII="/></pattern><path d="m.683472 6h29.0072v27.874h-29.0072z" fill="url(#a)"/><path d="m52.7904 15.96c-1.176-.588-2.856-1.036-4.536-1.036-3.78 0-5.936 2.744-5.936 6.692 0 3.836 2.016 6.608 5.936 6.608 1.932 0 3.472-.644 4.704-1.568l-.364-.616h-.308c-1.064.784-2.464 1.4-4.06 1.4-3.388 0-5.04-2.632-5.04-5.88s1.708-5.852 5.124-5.852c1.484 0 2.912.42 4.2 1.064h.28zm14.0775 10.892-.364-.588h-.308c-.896.756-2.072 1.176-3.248 1.176-1.54 0-2.436-.952-2.436-2.688v-16.688h-5.32v.756h4.452v15.82c0 2.156 1.008 3.584 3.304 3.584 1.484 0 2.772-.504 3.92-1.372zm14.0476-5.292c0-3.892-2.184-6.636-5.572-6.636s-5.572 2.744-5.572 6.636 2.128 6.664 5.544 6.664 5.6-2.772 5.6-6.664zm-.868 0c0 3.248-1.736 5.88-4.704 5.88s-4.704-2.632-4.704-5.88 1.792-5.852 4.704-5.852 4.704 2.604 4.704 5.852zm5.952-6.44v8.484c0 3.08 1.624 4.62 4.256 4.62 2.212 0 3.808-1.092 4.9-2.156l.308 1.96h.56v-12.908h-.868v10.192c-1.176 1.008-2.8 2.128-4.816 2.128-2.212 0-3.472-1.344-3.472-3.808v-8.512zm24.8235-7.056v8.568c-.784-.756-2.128-1.708-4.116-1.708-3.276 0-5.6 2.576-5.6 6.776 0 3.78 1.876 6.524 5.404 6.524 2.016 0 3.64-1.204 4.312-2.184l.308 1.988h.56v-19.964zm-4.032 7.644c1.68 0 2.996.84 4.032 1.764v7.784c-.84 1.008-2.268 2.184-4.2 2.184-3.136 0-4.648-2.632-4.648-5.768 0-3.668 2.072-5.964 4.816-5.964z"/></svg>
```

Comment:
it seems really weird to have an SVG with an embedded PNG, which means that the SVG won't behave like you'd typically expect, tho maybe this needs to be factored out between tilt + tilt-cloud

Reply:
Good catch! This was totally unintentional.